gap_analysis/models/business.py

```diff
@@ -0,0 +1,212 @@
+
+
+class BusinessEstimationLine(models.Model):
+    _name = "business.estimation.line"
+    _description = "Bus. Estimation Lines"
+
+    sequence = fields.Integer('Sequence')
+    name = fields.Char('Name', required=True)
+    business_requirement_id = fields.Many2one(
+        comodel_name='business.requirement',
+        string='Business Analysis',
+        ondelete='cascade'
+    )
+    type_id = fields.Many2one(
+        comodel_name='business.estimation.type',
+        string='Estimation Type',
+        ondelete='restrict'
+    )
+    estimated_time = fields.Integer(string='Estimated Time', required=True)
+    user_id = fields.Many2one(
+        comodel_name='res.users',
+        string='Assign To',
+        ondelete='set null'
+    )
+
+
+class BusinessRequirement(models.Model):
+    _name = "business.requirement"
+    _description = "Business Analysis"
+
+    @api.model
+    def _get_sequence(self):
+        sequence = self.env['ir.sequence'].get('business.requirement')
+        return sequence
+
+    sequence = fields.Char(
+        'Sequence',
+        readonly=True,
+        copy=False,
+    )
+    name = fields.Char(
+        'Name', required=True,
+        default=_get_sequence,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    description = fields.Char(
+        'Description', required=True,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    business_requirement = fields.Text(
+        'Customer Story',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    odoo_scenario = fields.Text(
+        'Odoo Scenario',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    odoo_gap = fields.Text(
+        'Odoo Gap',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    drop = fields.Boolean(
+        string='Drop?',
+        default=False,
+        readonly=True,
+        states={'draft': [('readonly', False)]},
+        help="""Determine whether this business requirement is to be kept
+        or dropped."""
+    )
+    rough_estimation_lines = fields.One2many(
+        comodel_name='business.estimation.line',
+        inverse_name='business_requirement_id',
+        string='Rough Estimation Lines',
+        copy=False,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    categ_id = fields.Many2one(
+        comodel_name='business.requirement.category',
+        string='Category',
+        ondelete='restrict',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    estimated_time_total = fields.Float(
+        compute='_get_estimated_time_total',
+        string='Total Estimated Time',
+        store=True,
+    )
+    state = fields.Selection(
+        selection="_get_states",
+        string='State',
+        default='draft',
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
```

Comment:
I like Stages as a configurable wrapper on top of static States. Maybe you can add that to the Roadmap section.